arviz/plots/backends/bokeh/kdeplot.py

```diff
@@ -226,10 +230,15 @@ def plot_kde(
                 palette=colors,
                 **pcolormesh_kwargs
             )
+            glyphs.append(image)
             ax.x_range.range_padding = ax.y_range.range_padding = 0
 
     if backend_show(show):
         bkp.show(ax, toolbar_location="above")
+
+    if legend and label is not None and return_glyph:
```

Comment:
the if should be only on return_glyph

Reply:
Then we are getting the problem of unused arguments `label` and `legend`.